java/java.lsp.server/vscode/package.json

```diff
@@ -464,22 +504,76 @@
 					"when": "nbJavaLSReady"
 				}
 			],
+			"view/title": [
+				{
+					"command": "java.workspace.newproject",
+					"when": "nbJavaLSReady && view == foundProjects",
+					"group": "navigation"
+				},
+				{
+					"command": "java.workspace.compile",
+					"when": "nbJavaLSReady && view == foundProjects"
+				},
+				{
+					"command": "java.clean.workspace",
+					"when": "nbJavaLSReady && view == foundProjects"
+				},
+				{
+					"command": "test-explorer.run-all",
+					"when": "nbJavaLSReady && view == foundProjects"
+				},
+				{
+					"command": "workbench.action.debug.run",
+					"when": "nbJavaLSReady && view == foundProjects"
+				},
+				{
+					"command": "workbench.action.debug.start",
+					"when": "nbJavaLSReady && view == foundProjects"
+				}
+			],
 			"view/item/context": [
 				{
 					"command": "foundProjects.deleteEntry",
 					"when": "view == foundProjects && viewItem == node"
 				},
 				{
 					"command": "nbls:Database:netbeans.db.explorer.action.Connect",
-					"when": "nbJavaLSReady && viewItem =~ /class:ddl.DBConnection/"
+					"when": "viewItem =~ /class:ddl.DBConnection/"
 				},
 				{
 					"command": "nbls:Database:netbeans.db.explorer.action.Disconnect",
-					"when": "nbJavaLSReady && viewItem =~ /class:ddl.DBConnection/"
+					"when": "viewItem =~ /class:ddl.DBConnection/"
 				},
 				{
 					"command": "nbls:Edit:org.openide.actions.DeleteAction",
-					"when": "nbJavaLSReady && viewItem =~ /class:ddl.DBConnection/"
+					"when": "viewItem =~ /cap:delete/"
```

Comment:
What does `=~ /cap:delete/` mean?

Reply:
I guess this is another 'private API' that needs a documentation. 

We had defined `class:something` to represent services on the node; I've "defined" `cap:` as prefix for capabilities (canRemove, canDestroy (well, delete), canRename -- possibly we'll find more) - neither of these is represented by a type, so different prefix to avoid possible clash. And yet `is:folder` -- I could turn that one into `class:DataFolder` (although it not exactly true, but could be better than yet another magic selector).

Reply:
Covered in d56b619